### ISSUE TITLE
Customize CSS for admonitions and admonitions block

### ISSUE THREAD

**Issue by prudhomm** [feelpp/feelpp-book#30]
@gdolle could you customize the CSS for admonitions and admonitions blocks in see http://asciidoctor.org/docs/asciidoc-syntax-quick-reference/

it would be nice to have something like in the asciidoc reference.

**Comment by gdolle** [comment]
Ok I'll look into that.
https://github.com/GitbookIO/gitbook/issues/1430
Also we've already richquotes, if we use asciidoc features, I propose to remove richquotes.

**Comment by prudhomm** [comment]
yes I agree. asciidoc supports also block admonitions and it is rendered in the book as well. I think a little bit of CSS and we should have something nice
